cypress/e2e/dashboard_atomic_tests.cy.js

```diff
@@ -0,0 +1,185 @@
+import { HomePage } from "../page-objects/home_page";
+import { LoginPage } from "../page-objects/login_page";
+import { faker } from "@faker-js/faker";
+
+describe("Dashboasrd atomic tests", { testIsolation: false }, () => {
+  before(() => {
+    cy.clearAllCookies();
+    cy.clearAllLocalStorage();
+    cy.clearAllSessionStorage();
+    const username = faker.internet.userName();
+    const password = faker.internet.password({ length: 15 });
+    const email = faker.internet.exampleEmail();
+    new LoginPage()
+      .openTegb()
+      .clickregisterNewUser()
+      .typeNewUsername(username)
+      .typeNewPassword(password)
+      .typeNewEmail(email)
+      .clickRegister();
+    new LoginPage()
```

Comment:
Proč tady voláš novou LoginPage? To se přeci dalo udělat v rámci jednoho FluentAPI příkazu